### ISSUE TITLE
Why do the resources not implement Properties?

### ISSUE THREAD

**Issue by NiklasEi** [guimcaballero/bevy_chess#3]
> Easy as that! We're deriving Default so that when the plugin is initialized it starts with a None value, but we could provide an initial value in case we wanted by implementing FromResources. You can see how it works in this Bevy example.

The liked example in the last sentence states
> Registered components must implement the `Properties` and `FromResources` traits.`

Aren't our resources missing `Properties`? Do we need it? If not, why not?

**Comment by NiklasEi** [comment]
Ah, I think I confused Resources and Components here. In our case, we have a resource and you talk about implementing `FromResources`, but in the linked bevy example `FromResources` is discussed for `Components`. Could you please shed some light on that?

**Comment by guimcaballero** [comment]
Looks like I linked the wrong example! That one demonstrates implementing `FromResources` for a `Component`, while in the tutorial I was talking about implementing it for a `Resource`.  I haven't really worked much with implementing `FromResources` and `Properties` for a `Component`, so I can't really tell how they compare.

It has been fixed, it now links to a different (and now correct) example. Thanks for catching that!
